### PR TITLE
Raz Allow to abort installation in wrong boot order …

### PR DESCRIPTION
**Pull request by RazRegev**

### CODE REVIEW

File: subsystem/cluster_test.go [hunk 1]
```diff
@@ -849,6 +849,34 @@ var _ = Describe("cluster install", func() {
...
+				host := getHost(clusterID, *hostID)
+				_, err = bmclient.Installer.RegisterHost(ctx, &installer.RegisterHostParams{
+					ClusterID: clusterID,
+					NewHostParams: &models.HostCreateParams{
+						HostID: host.ID,
+					},
+				})
+				waitForHostState(ctx, clusterID, *host.ID, models.HostStatusInstallingPendingUserAction,
+					defaultWaitForHostStateTimeout)
+				_, err = bmclient.Installer.CancelInstallation(ctx, &installer.CancelInstallationParams{ClusterID: clusterID})
```
Comment (eranco74) [inline subsystem/cluster_test.go:870]: Add another test that first cancel and than the host re-register (and after validating host status reset)

Comment (RazRegev) [reply subsystem/cluster_test.go:871]: There is already a test that validate host can register after cancel+reset

Comment (eranco74) [reply subsystem/cluster_test.go:871]: I think we need a test that first cancel and than the host re-register (prior to the reset)

File: subsystem/cluster_test.go [hunk 2]
```diff
@@ -849,6 +849,35 @@ var _ = Describe("cluster install", func() {
 					Expect(swag.StringValue(host.Status)).Should(Equal(models.HostStatusError))
 				}
 			})
+			It("[only_k8s]cancel installation - wrong boot order", func() {
```
Comment (filanov) [inline subsystem/cluster_test.go:852]: it's a long test, please add steps `By(..)`

**Pull request closed without merging**